mod/storage/pkg/deposit/store.go

```diff
@@ -115,8 +170,20 @@ func (kv *KVStore[DepositT]) Prune(start, end uint64) error {
 	for i := range end {
 		// This only errors if the key passed in cannot be encoded.
 		if err := kv.store.Remove(ctx, start+i); err != nil {
+			kv.logger.Error(
```

Comment:
Note: the pruned is completelly wrong. It seems to return `(start, range)` instead of `(start, end)`.
Will fix in a different PR as soon as this fix is unlocked.
I guess the pruner issue was kinda silent since we don't really stop the node when it happens, just log an error

Reply:
Do we have GH issue for this? Not sure, maybe its related to https://github.com/berachain/beacon-kit/issues/2134